src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+  const myPerson = {
+    name: name,
```

Comment:
When the property key and value name are the same you can use a shorthand:
````
const createPerson = (name, age) => {
  return {
    name,
    age
  };
};
`````
Also because you do not use myPerson anywhere else, only at the return statement you can just return the object straightaway instead of assigning it to a variable :)